.github/workflows/style.yml

```diff
@@ -0,0 +1,13 @@
+name: Style
+
+on: [push, pull_request]
+
+jobs:
+  style:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Tabs
+        run: .github/workflows/style/check_tabs.sh
+      - name: Trailing Whitespaces
```

Comment:
@WeiqunZhang If the tabs check fails, it will not run trailing whitespaces check. Probably want to split this into two independent jobs?

Reply:
Sure, we can split this.